content/contributing-code/contents.lr

```diff
@@ -32,17 +34,21 @@ Some helpful saved searches on GitHub than can assist with finding an issue:
 
 Once you've found an issue you'd like to work on, please follow these steps to make your contribution:
 
-1. Comment on it and say you'd like to work on that issue.
-2. Wait for someone (usually a CC staff member or project maintainer) to confirm that you may work on the issue before writing any code. The person who confirms will assign the issue to you add an "in progress" label to the issue to indicate that the issue is being worked on.
-3. Once the issue has been updated to "in progress", write your code and submit your pull request. Be sure to read and follow our **[pull request guidelines](/contributing-code/pr-guidelines)**!
-4. Wait for code review and address any issues raised as soon as you can.
+1. Comment on it and say you're working on that issue. This is to avoid conflicts with others also working on the issue.
+    * A project maintainer may mark the issue with an `in progress` label at this point, but we don't always get around to that.
+2. Write your code and submit your pull request. Be sure to read and follow our **[pull request guidelines](/contributing-code/pr-guidelines)**!
+    * Please link to your branch or PR in the issue as soon as you push it, even if it's only an early draft.
```

Comment:
If they do the `Fixes #<issue_number` (which they should) it automatically gets linked in the issue. It might be worth creating a `WIP` label on the repos and ask people to open the PR as soon as the first commit gets pushed and then mark the PR with the WIP label. Once the work is done they could remove the WIP label and mark reviewers